### DISCUSSION TITLE
Very big delay between calls

### DISCUSSION THREAD

**Discussion by integral-llc** [dustinchilson/NLog.Targets.GraylogHttp#57] [Q&A] [answered]
I did install a basic Graylog server, based on the Graylog AMI  on AWS, created tge simplest C# app with the such a content 
```

static void Main(string[] args)
        {
          var logger = LogManager.GetCurrentClassLogger();
          logger.Trace("String Trace");
          logger.Debug("String Debug");
          logger.Warn("String Warn");
          logger.Error("String Error");
          logger.Fatal("String Fatal");

          var e = new LogEventInfo(LogLevel.Fatal, "Test", "Test Messagewith Prop");
          e.Properties["test_prop"] = "test property";
          logger.Log(e);

          Console.WriteLine("Completed");
          Console.ReadKey();
        }
```

The logs are properly being logged in my graylog server, BUT, it takes about 1 minute for this program to complete, is like the HTPP call takes forever to complete and the app just waits. 

Any idea what this can be?

**Comment by dustinchilson** [comment]
What does your nlog config look like in the app.config.

**Comment by integral-llc** [comment, marked as answer]
it looks like this is a known issue in 3.0.0.  https://github.com/Graylog2/graylog2-server/pull/5728

Waiting for the 3.0.1 to be released, sorry to bug you :)

**Comment by dustinchilson** [comment]
Oh good to know. I haven't upgraded to 3.0 yet.
